src/Core/Services/AuthService.cs

```diff
@@ -222,10 +223,13 @@ public async Task<AuthResult> LogInPasswordlessAsync(string email, string access
                 return null;
             }
 
+            // The approval device may not have a master key hash if it authenticated with a passwordless method
             if (string.IsNullOrEmpty(masterKeyHash) && decryptionKey != null)
             {
+                var authResult = await LogInHelperAsync(email, accessCode, null, null, null, null, null, null, null, null, null, authRequestId: authRequestId);
```

Comment:
♻️ Just putting it here to make a note, we should definitely look into refactoring `LogInHelperAsync` to avoid this type of usage, it has too many arguments. Maybe for a tech-debt task given that it doesn't affect this PR only, but several other flows as well.